Aptoide/aptoide-client-v8
# [MOB-82] Implement Sentry

**What does this PR do?**

This PR aims at implementing sentry. It uses different keys for dev, cobrand and prod release (these conditions can be confirmed in the build gradle file).
Note that for this, new keys were added and also, REALLY IMPORTANT !!!! , there is a new file that should be added to the project. Talk to me for more updates on it.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

Open the app, find a way to crash it (either you change the code and throw an exception) or just try to use any known ways to make our app crash :P. Check if everything is ok with the logs and then talk to me so that we can see the crash in the sentry dashboard (if you want). 

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-82](https://aptoide.atlassian.net/browse/MOB-82)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 
Yes new keys for sentry from debug cobrand and release.
Also new sentry properties file that should be in the other repo.


**Code Review Checklist**
